src/BasketBundle/Resources/config/basket.xml

```diff
@@ -24,10 +24,14 @@
         <service id="sonata.basket" class="Sonata\Component\Basket\Basket" public="true">
             <factory service="sonata.basket.loader" method="getBasket"/>
         </service>
-        <service id="sonata.basket.block.nb_items" class="Sonata\BasketBundle\Block\BasketBlockService">
+        <service id="sonata.basket.block.nb_items" class="Sonata\BasketBundle\Block\BasketBlockService" public="true">
```

Comment:
Why does this service need to be public? If there is a good reason, why not contribute it to the stable branch? It's not a BC break.

Reply:
no good reason, maybe wrong file updated when looked after the add basket issue